### DISCUSSION TITLE
Redundant/backup RTMP URL stream

### DISCUSSION THREAD

**Discussion by veks-streamer** [MonaSolutions/MonaServer2#54] [Q&A]
Hi,
I'm trying to figure out if there's a way to have a second backup RTMP ingest URL, something like YouTube, Restream.io has.

The idea is to stream primary stream to something like **rtmp://10.10.50.10/live/test** and the backup to **rtmp://10.10.50.10/live/test?bkp** and if first stream stops for some reason, disconnects, then Mona should switch to the bkp stream and continue ingesting.
And if primary goes back up, switch to it instantly.

Is there a way to achieve that with MonaServer?
Thanks!

**Comment by thomasjammet** [comment]
Hi!

There is no such feature for now, though you can implement the switch on the publisher side. 
For example if the RTMP publisher restart a new publication this can be done instantly and should be transparent for the players.
Otherwise you can also switch stream on the player side when receiving the end of stream notification.

**Reply by veks-streamer** [reply]
Sadly this aint the solution. Because stream shouldn't be interrupted. 
Is there a another way to achieve this? 
Or at least have two separate inputs that can be restreamed somehow to another output, but somehow can be selected which input will go to the output? Inside the monaserver.
Tnx!
